packages/optionmenu/Option.svelte

```diff
@@ -21,21 +21,21 @@
   // Just a little more readable
   const enabled = !disabled;
 
-  // If the enabled status of this option changed, update the enabledOptions
-  // store
-  $: if (enabled && !enabledOptions.has(key)) {
-    enabledOptions.add(key, label);
-  } else if (!enabled && enabledOptions.has(key)) {
-    enabledOptions.remove(key);
-  }
+  enabledOptions.subscribe(() => {
+    if (enabled && !enabledOptions.has(key)) {
+      enabledOptions.add(key, label);
+    } else if (!enabled && enabledOptions.has(key)) {
+      enabledOptions.remove(key);
+    }
+  });
 
-  // If this option is enabled the selected status changed, update the
-  // selectedOptions store
-  $: if (enabled && selected && !selectedOptions.has(key)) {
-    selectedOptions.add(key, label);
-  } else if (enabled && !selected && selectedOptions.has(key)) {
-    selectedOptions.remove(key);
-  }
+  selectedOptions.subscribe(() => {
+    if (enabled && selected && !selectedOptions.has(key)) {
+      selectedOptions.add(key, label);
+    } else if (enabled && !selected && selectedOptions.has(key)) {
+      selectedOptions.remove(key);
+    }
+  });
```

Comment:
This seems circuitous and probably inefficient. I suspect there's an easier way to repopulate the enabled/selected options list than this.

Reply:
This just rewrites this logic to use `subscribe` instead of the `$:`? Or is there more to it than that?

Reply:
Hmm,, I believe the `$:` doesn't actually detect that the contents of selectedOptions changed, so is ineffective.

Reply:
Okay. I'm going to approve this to not block this further.